packages/renderer-protocol/inverseRpc/services/friendRequestService.ts

```diff
@@ -59,7 +60,52 @@ export function registerFriendRequestKernelService(port: RpcServerPort<RendererP
     },
 
     async sendFriendRequest(req, _) {
-      return {}
+      try {
+        // Handle send friend request
+        const sendFriendRequest = await requestFriendship(req)
+
+        let sendFriendRequestReply: SendFriendRequestReply = {}
+
+        // Check the response type
+        if (sendFriendRequest.reply?.friendRequest) {
+          sendFriendRequestReply = {
+            message: {
+              $case: 'reply',
+              reply: {
+                friendRequest: {
+                  friendRequestId: sendFriendRequest.reply.friendRequest.friendRequestId,
+                  timestamp: sendFriendRequest.reply.friendRequest.timestamp,
+                  to: sendFriendRequest.reply.friendRequest.to,
+                  from: sendFriendRequest.reply.friendRequest.from,
+                  messageBody: sendFriendRequest.reply.friendRequest.messageBody
+                }
+              }
+            }
+          }
+        } else {
+          sendFriendRequestReply = {
+            message: {
+              $case: 'error',
+              error: sendFriendRequest.error ?? FriendshipErrorCode.FEC_UNKNOWN
+            }
+          }
```

Comment:
How about a util function that constructs this object with an unknown error by default?

Reply:
This is the best I can do ([link](https://github.com/decentraland/kernel/pull/760/commits/025d216fdec4fe3b245ab093d92739661d77f049)). If I try to extract this into a more generic func:
```
{
    message: {
      $case: 'error',
      error: error ?? FriendshipErrorCode.FEC_UNKNOWN
    }
}
```
 I get the error:
```
Types of property 'message' are incompatible.
Type '{ $case: string; error: FriendshipErrorCode; }' is not assignable to type '{ $case: "error"; error: FriendshipErrorCode; }'.
 ```

Reply:
It's a bit trickier since the response interfaces are defined for the reply and error formats as a single type. [But here you can find an example of how to unify them](https://github.com/decentraland/kernel/pull/784).